spec/spec_helper.rb

```diff
@@ -10,3 +10,22 @@ module SpecHelper
 end
 
 WebMock.disable_net_connect!(allow: 'coveralls.io')
+
+# Executes the provided block after adjusting the ENV to have the
+# provided keys and values set as defined in hash. After the block
+# completes, restores the ENV to its previous state.
+def with_env_values(hash)
+	begin
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Redundant `begin` block detected.<br>Tab detected.